src/scenes/Jarjestajat/Jarjestaja/Hakemukset/Muutospyynto/components/MuutospyyntoWizardOpiskelijavuodet.js

```diff
@@ -79,15 +78,16 @@ const MuutospyyntoWizardOpiskelijavuodet = React.memo(props => {
     const maarays = R.find(R.propEq("koodisto", "koulutussektori"))(
       props.maaraykset
     );
-    if (maarays) {
-      setKoodiarvot(prevState => {
-        return {
-          ...prevState,
-          vahimmaisopiskelijavuodet: maarays.koodiarvo
-        };
-      });
-    }
-  }, [props.maaraykset]);
+    setKoodiarvot(prevState => {
+      return {
+        ...prevState,
+        vahimmaisopiskelijavuodet: {
+          koodiarvo: maarays ? maarays.koodiarvo : "3", // 3 = Ammatillinen koulutus
+          maaraysUuid: getKohde("Ammatillinen koulutus", opiskelijavuodet).maaraysUuid
```

Comment:
Does "Ammatillinen koulutus" work also when locale isn't Finnish?

Reply:
Works currently in swedish. Maybe because section 5 titles are not localized.
Copied logic from existing original value states (setInitialValue, setInitialValueSisaoppilaitos, setInitialValueVaativa).

All this logic should be refactored.